ui/filewriter_ctrl_frame.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'filewriter_command_frame.ui',
+# licensing of 'filewriter_command_frame.ui' applies.
+#
+# Created: Mon Nov 11 16:21:42 2019
+#      by: pyside2-uic  running on PySide2 5.13.0
+#
+# WARNING! All changes made in this file will be lost!
+
+from PySide2 import QtCore, QtGui, QtWidgets
+
+class Ui_FilewriterCtrl(object):
+    def setupUi(self, FilewriterCtrl):
+        FilewriterCtrl.setObjectName("FilewriterCtrl")
```

Comment:
i think this could do with refactoring into simple methods that are responsible for setting up certain parts of the UI.